src/routes/articles/assignTagToArticle.e2e.test.js

```diff
@@ -0,0 +1,140 @@
+const {expect} = require('chai')
+const {fixtures} = require('../../../test/utils')
+
+
+describe('POST /articles/:id/tags', () => {
+
+	let user
+	let accessToken
+
+	before(async ()=>{
+
+		 user = await fixtures.insertUser() 
+		const body = { id: user.id, email: user.email }
+		accessToken = fixtures.generateAccessToken(
+			{user : body}
```

Comment:
Let us kindly fix the issues with lining here as well. 
Either we do not have the linter set well in the project.
For instance extra lines, and spaces, etc